CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [2.6.1] - 2020-11-09
```

Comment:
```suggestion
## [2.7.0] - 2020-11-09
```

Nitpick but can we make this a minor rather than a patch? As I think its new functionality rather than a fix.

Reply:
Agreed. I'll create a new branch for 2.7.0.